satellite/console/consoleweb/server.go

```diff
@@ -444,3 +472,66 @@ func (server *Server) gzipHandler(fn http.Handler) http.Handler {
 		fn.ServeHTTP(w, newRequest)
 	})
 }
+
+// initializeTemplates is used to initialize all templates
+func (server *Server) initializeTemplates() (err error) {
+	server.templates = make(map[consoleTemplate]*template.Template)
+
+	notFoundErr := server.parseTemplate(notFound, path.Join(server.config.StaticDir, "static", "errors", "404.html"))
```

Comment:
There is `parseGlob` method for parsing group of templates at once, maybe you can use it here